examples/vault-consul-ami/vault-consul.json

```diff
@@ -52,11 +52,11 @@
   }],
   "provisioners": [{
     "type": "shell",
-    "inline": ["mkdir /tmp/terraform-aws-vault"]
+    "inline": ["mkdir -p /tmp/terraform-aws-vault/modules"]
```

Comment:
This bit reduced my upload when building the amis when testing locally from ~650MB to ~200KB because it avoids uploading `.terraform` dirs and `test/vendor`